### DISCUSSION TITLE
手贱点了一下开始抢票，结果控制台里显示账户异常，这个有关系么

### DISCUSSION THREAD

**Discussion by Vanadium9891** [mikumifa/biliTickerBuy#199] [提问专区]
“您的账户存在异常，暂时无法购票”
真的好担心啊
![IMG_20240629_103034](https://github.com/mikumifa/biliTickerBuy/assets/140573321/886f7bd5-f626-4c73-a2b5-7a370c9f27d8)

**Comment by mikumifa** [comment]
建议换号

**Comment by mikumifa** [comment]
这种特别火的非法请求可能会封

**Reply by Vanadium9891** [reply]
可是这个号有大会员啊

**Reply by mikumifa** [reply]
![Image_1719629425398.jpg](https://github.com/mikumifa/biliTickerBuy/assets/99951454/5965087e-b280-4482-acde-f08f550b808d)

**Reply by mikumifa** [reply]
先账号看看买其他票

**Reply by YourComputer011** [reply]
> ![Image_1719629425398.jpg](https://private-user-images.githubusercontent.com/99951454/344361473-5965087e-b280-4482-acde-f08f550b808d.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MzA5NDMsIm5iZiI6MTcxOTYzMDY0MywicGF0aCI6Ii85OTk1MTQ1NC8zNDQzNjE0NzMtNTk2NTA4N2UtYjI4MC00NDgyLWFjZGUtZjA4ZjU1MGI4MDhkLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDAzMTA0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4Y2Y0ODc5YWE1NjY0M2Q4YjJiZmE0MTc1YmQ3NDgwMjlkZTJmMDk5MWUwNjNmMjk0NjM4NmJiNGM5NTNjNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kU5Bym6GxbsSDvS8GscPWxn6doLaPaPKdHLJ4iuxEJI)

话说这样的话【间隔】和【开始抢票时间】的设置是不是都要留出一些余地了

**Reply by mikumifa** [reply]
大胆设

**Reply by mikumifa** [reply]
不提前就行

**Reply by drinknaixi-1** [reply]
之前看到帖子说计时器有延迟，最好提前设，那么现在要怎么办呢

**Reply by mikumifa** [reply]
提前一分钟开始定时抢票，时间是设12点，
别提前几小时去设置12点

定时器确实可能有延迟，尽量不要让定时器定太长时间
